lib/av_client/crypto/pedersen_commitment.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  addPoints,
+  Bignum,
+  bignumFromHex,
+  bignumToHex,
+  Curve,
+  generateRandomBignum,
+  hashToBignum,
+  Point,
+  pointFromHex,
+  pointFromX,
+  pointToHex,
+} from "./util";
+
+class PedersenCommitment {
+  static computeGenerator(index) {
+    const baseGeneratorPrefix = () => pointToHex(new Point(Curve.G));
+    const secp256k1_curve_prime = new Bignum(Curve.field.modulus);
+
+    const target = [baseGeneratorPrefix(), index].join(',');
```

Comment:
Just a question: what is `target`? I can see it contains a string with a comma (and probably numbers on each side), but it is unclear why this is a good idea.